code/_compile_options.dm

```diff
@@ -70,4 +70,8 @@
 #endif
 
  #define SERVICE_CMD_PARAM_KEY "serviceCommsKey"
- 
\ No newline at end of file
+
+//Update this whenever the db schema changes 

```

Comment:
oh wait, its the name of key in a key value pair.

`serviceCommsKey` is just some key for an associated array, its not the actual coms key, its the array key for the coms key (`var/comskey = href_list["serviceCommsKey"]`)